dds_web/security/tokens.py

```diff
@@ -29,7 +29,13 @@ def encrypted_jwt_token(
     :param Dict or None additional_claims: Any additional token claims can be added. e.g., {"iss": "DDS"}
     """
     token = jwt.JWT(
-        header={"alg": "A256KW", "enc": "A256GCM"},
+        header={
+            "alg": "A256KW",
+            "enc": "A256GCM",
+            "exp": (dds_web.utils.current_time() + expires_in).timestamp(),
+            "iat": (dds_web.utils.current_time()).timestamp(),
```

Comment:
Would be great if we could att clarification comments regarding what the different headers stand for, for the future